src/integrationtest/StatementExecution.test.ts

```diff
@@ -186,12 +187,12 @@ describe("StatementExecution", function() {
 
         const searchQuery: string = `SELECT VALUE ${constants.COLUMN_NAME} FROM ${constants.TABLE_NAME} WHERE ` + 
             `${constants.COLUMN_NAME} IN (?,?)`;
-        const tables: string[] = await driver.executeLambda(async (txn: TransactionExecutor) => {
+        const domValues: dom.Value[] = await driver.executeLambda(async (txn: TransactionExecutor) => {
             const result: Result = await txn.execute(searchQuery, constants.MULTI_DOC_VALUE_1, constants.MULTI_DOC_VALUE_2);
-            const resultSet: dom.Value[] = result.getResultList();
-            return resultSet.map((value: dom.Value) => {
-                return value.stringValue();
-            });
+            return result.getResultList();
+        });
+        const tables: string[] = domValues.map((value: dom.Value) => {
```

Comment:
Is it not possible to do this mapping within the lambda? If there are no elements then the returned set would be empty so the map would simply operate on no elements which would be fine, or is that incorrect?

Reply:
Problem is that it may in some cases return a null ion value, which will error out if you try to call stringvalue on it.